fuels-abigen-macro/Cargo.toml

```diff
@@ -15,14 +15,14 @@ proc-macro = true
 Inflector = "0.11"
 anyhow = "1"
 bytes = { version = "1.0.1", features = ["serde"] }
-forc = { version = "0.2", features = ["test", "util"], default-features = false }
+forc = { version = "0.3", features = ["test", "util"], default-features = false }
 fuels-core = { version = "0.1.3", path = "../fuels-core" }
 fuel-asm = { version = "0.1", features = ["serde-types"] }
-fuel-gql-client = { version = "0.1", default-features = false }
-fuel-core = { version = "0.1", default-features = false }
-fuel-tx = "0.1"
+fuel-gql-client = { version = "0.2", default-features = false }
+fuel-core = "0.2"
```

Comment:
Shouldn't remove `default-features = false` here and in the other manifest file.